scripts/html_chunking/chunker.py

```diff
@@ -68,13 +68,17 @@ def chunk_html(
         count_tag_tokens=count_tag_tokens
     )
 
+    quick_link = kwargs.get('quick_link', False)
     try:
         soup = BeautifulSoup(html_content, 'html.parser')
         document_title = get_document_title(soup)
 
         if count_html_tokens(html_content, options.count_tag_tokens) <= options.max_token_limit:
+            docs_url = source_url
+            if (section := soup.find('section')) and section.get('id'):
+                docs_url = _get_anchored_url(source_url, section.get('id'), quick_link=quick_link)
```

Comment:
I'm figuring we're handling a document that fits into a single chunk. Is a document going to have a section/id for this branch to make sense?

Reply:
Fair enough, I just had it in to be same across the board wherever we are generating link.

I'll drop this now that I don't see any added value.